e2e/tests/008.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set the new value for maxUplinkThroughput as a parameter
+
+set -o pipefail
+set -o errexit
+set -o nounset
+[[ "${DEBUG:-false}" != "true" ]] || set -o xtrace
+
+new_capacity_value=$1
+
+export HOME=${HOME:-/home/ubuntu/}
+export E2EDIR=${E2EDIR:-$HOME/test-infra/e2e}
+export TESTDIR=${TESTDIR:-$E2EDIR/tests}
+export LIBDIR=${LIBDIR:-$E2EDIR/lib}
+
+source "${LIBDIR}/k8s.sh"
+
+kubeconfig="$HOME/.kube/config"
+
+#upf_deployment_pkg=$(kpt alpha rpkg get --name free5gc-packages  --revision v3 -o jsonpath='{.metadata.name}')
+
+#upf_pkg_rev=$(kpt alpha rpkg clone -n default "$upf_deployment_pkg" --repository mgmt edge01 | cut -f 3 -d ' ')
+
+upf_deployment_pkg=$(kpt alpha rpkg get --name free5gc-upf | grep edge01 | grep latest | cut -d ' ' -f 1)
```

Comment:
So, rather than start with the package in the free5gc-upf repo, we want to find the currently running variant of that package. This will live in the edge01 repo instead.